workflows/rnaseq-ref-index/build-index.nf

```diff
@@ -2,9 +2,11 @@
 nextflow.enable.dsl=2
 
 // basic parameters
-params.ref_dir = 's3://nextflow-ccdl-data/reference/homo_sapiens/ensembl-100'
+params.ref_dir = 's3://nextflow-ccdl-data/reference/homo_sapiens/ensembl-103'
 params.cdna = 'fasta/Homo_sapiens.GRCh38.cdna.all.fa.gz'
```

Comment:
We should probably remove this; I don't think we are likely to use a cdna-only index given earlier testing. We almost certainly want the ncRNA!